app/renderer/components/Inspector/Screenshot.js

```diff
@@ -166,4 +165,4 @@ class Screenshot extends Component {
   }
 }
 
-export default withTranslation(Screenshot);
```

Comment:
is the wrapper not necessary anymore?

Reply:
I didn't see it doing anything, and it was preventing me from accessing the component from another component, so I removed it.

Reply:
It was needed for localization to work. Does it still look ok if you change the app language?

Reply:
oh, I didn't try. I just saw that the component didn't seem to have any text in it so I figured it was an extraneous i18n wrapper. but i don't want to break it if it is doing something

Reply:
i was able to refactor this PR so that i could replace the `withTranslation` wrapper.